pkg/apis/networking/validation/validation.go

```diff
@@ -68,11 +68,21 @@ func ValidateNetworkPolicyPort(port *networking.NetworkPolicyPort, portPath *fie
 			for _, msg := range validation.IsValidPortNum(int(port.Port.IntVal)) {
 				allErrs = append(allErrs, field.Invalid(portPath.Child("port"), port.Port.IntVal, msg))
 			}
+			if port.EndPort != nil && *port.EndPort < port.Port.IntVal {
+				allErrs = append(allErrs, field.Invalid(portPath.Child("endPort"), port.Port.IntVal, "endPort field must be equal or greater than port field"))
```

Comment:
validation errors already include the field name (that is what `port.Path.Child("endPort")` is for) - don't repeat it.

Reply:
conventions say:

```
"must be greater than or equal to `port`"
```